distributionviewer/core/static/css/distributionviewer.styl

```diff
@@ -33,19 +38,15 @@ a {
 }
 
 .filters {
-  background-color: $green;
-  margin-right: $filters-margin-right;
-  padding-left: 40px;
-  padding-right: @padding-left;
-  min-width: $filters-width;
-
-  /* Make the filter sidebar touch the top of the page, but keep the text
-     aligned with the main content. */
-  margin-top: -($header-height)
-  padding-top: $header-height;
+  position: absolute;
+  width: $filters-width;
+  top: 10px;
```

Comment:
Since this value should be equal to the `padding-top` of the `global-wrapper`, we could consider putting it into a variable.